src/lib/draw-and-pick.js

```diff
@@ -351,7 +376,13 @@ function getPickedColors(gl, {
             viewport: layer.context.viewport
           }),
           uniforms: Object.assign(
-            {renderPickingBuffer: 1, pickingEnabled: 1},
+            // TODO: Update all layers to use 'picking_uActive' (picking shader module)
+            // and then remove 'renderPickingBuffer' and 'pickingEnabled'.
+            {
+              renderPickingBuffer: 1,
+              pickingEnabled: 1,
+              picking_uActive: 1
```

Comment:
> renderPickingBuffer and pickingEnabled do not belong to picking module

Very good point, we can deprecated them, but they will be used by existing custom layers so we need to keep supporting them at least until we make a major release.

Ideally we'd emit warnings when layers use these uniforms in their shaders. There are ways we can check what uniforms layer shaders define. But not needed for this PR, better open a separate issue: "deprecate old picking uniforms and updating to use new picking module in all sample layers".